synapseclient/table.py

```diff
@@ -2445,14 +2470,23 @@ def _update_self(self, syn):
             self.etag = upload_to_table_result["etag"]
         return self
 
-    def asDataFrame(self, rowIdAndVersionInIndex=True, convert_to_datetime=False):
+    def asDataFrame(
+        self,
+        rowIdAndVersionInIndex=True,
+        convert_to_datetime=False,
+        na_values=None,
+        keep_default_na=True,
```

Comment:
Does it make sense to take a kwargs here to support all pd.Dataframe kwargs?

Reply:
I don't see why not. I'll add this